extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -57,8 +62,15 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
 
     /** @private {!AmpElement} element */
     this.shoppingTagEl_ = null;
+
     /** @private {?Element} */
     this.pageEl_ = null;
+
+    /** @private @const {!./story-analytics.StoryAnalyticsService} */
+    this.analyticsService_ = getAnalyticsService(
+      this.win,
+      ancestorElementsByTag(this.element, 'amp-story')[0]
+    );
```

Comment:
Please use the getter from Services like we do for the store and localization services.
There is an example in [amp-story-page-attachment](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story-page-attachment/0.1/amp-story-page-attachment.js#L85-L86)